recipe/meta.yaml

```diff
@@ -36,7 +36,8 @@ requirements:
     - matplotlib >=1.3.1
     - networkx >=1.8
     - pillow >=2.1.0
-    - dask >=0.5
+    - dask-core >=0.15
+    - toolz >=0.8.2
```

Comment:
Should only need [`toolz >= 0.7.3`]( https://github.com/dask/dask/blob/0.15.0/setup.py#L10 ) for `dask >=0.15` unless `scikit-image` has its own requirements for `toolz`.